packages/flutter/lib/src/cassowary/param.dart

```diff
@@ -6,45 +6,66 @@ import 'equation_member.dart';
 import 'expression.dart';
 import 'term.dart';
 
+/// Represents a variable inside the layout solver.
 class Variable {
   static int _total = 0;
 
+  /// Create a new variable with the given constant value.
   Variable(this.value) : _tick = _total++;
 
   final int _tick;
 
+  /// The current value of the variable.
   double value;
 
+  /// An optional name given to the variable. This is useful in debugging
+  /// solver state.
   String name;
 
+  /// Variables represent state inside the solver. This state is usually of
+  /// interest to some entity outside the solver. Such entities can (optionally)
+  /// associate themselves with these variables. This means that when solver
+  /// is flushed, it is easy to obtain a reference to the entity the variable
+  /// is associated with.
```

Comment:
but _owner is used by Param, which seems to be "inside" the solver, unless i misunderstand... It's not clear how you would set `owner` if you're not a Param.